lib/queryContextCache.js

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
```

Comment:
should it be 2023?

Reply:
Yes. Fixed, and added the copy right on the new files.